src/hermes_types.h

```diff
@@ -46,11 +46,22 @@ constexpr char kPlaceInHierarchy[] = "PlaceInHierarchy";
 #define HERMES_NOT_IMPLEMENTED_YET \
   LOG(FATAL) << __func__ << " not implemented yet\n"
 
+union TargetID {
+  struct {
```

Comment:
Can we have an explanation here how to understand this struct as a `TargetID`?

Reply:
I added some documentation. Does that answer your question?

Reply:
Yes, thanks!